config/sidekiq.yml

```diff
@@ -7,7 +7,7 @@
   - default
   - mailers
 
-:schedule:
+:scheduler:
```

Comment:
Reading the changing PR, it looks like we should move the `schedule` key under a new `scheduler` key, not just rename it. I will do that change.

Reply:
oh ok, I must have misread. Thanks!